aws/data_source_aws_instance.go

```diff
@@ -316,6 +316,21 @@ func dataSourceAwsInstance() *schema.Resource {
 				Type:     schema.TypeBool,
 				Computed: true,
 			},
+			"enclave_options": {
+				Type:     schema.TypeList,
+				Optional: true,
```

Comment:
Since this schema attribute cannot be used to filter the lookup as an argument, `Optional` should be removed. 👍 

```suggestion
```